sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -269,12 +271,14 @@ export class ServiceBusClient {
    * https://docs.microsoft.com/azure/service-bus-messaging/message-transfers-locks-settlement#peeklock
    *
    * @param queueName The name of the queue to receive from.
+   * @param sessionId The id of the session from which messages need to be received
    * @param options Options include receiveMode(defaulted to peekLock), options to create session receiver.
    * @returns A receiver that can be used to receive messages of the form `ServiceBusReceivedMessageWithLock`
    */
-  createSessionReceiver(
+  acceptSession(
```

Comment:
It is worth updating the docs here regarding what exactly are we "accepting". Can be done in a separate PR

Reply:
Are you thinking of something at the method name level or just more documentation?

Reply:
Am thinking of the docs for the `acceptSession` and `acceptNextSession` methods